ref/src/sw_driver.F90

```diff
@@ -73,15 +93,18 @@ program sw_driver
   ! Read the debug settings from the namelist
   read(nl_unit, nml=debug)
 
+  ! Read the MPI decomposition from the namelist
+  read(nl_unit, nml=decomposition)
```

Comment:
Should this be inside of a `#ifdef ENABLE_MPI` ?

Reply:
@christopherwharrop-noaa It could be but as it is it's in the spirit of always calling GPTL init for simplicity.

Reply:
I'm not following you.  Why would we need to read the `decomposition` namelist if we're not using MPI?  What if the namelist doesn't have that section because someone is not using MPI?

Reply:
I'm not following you. Why would we need to read the decomposition namelist if we're not using MPI? What if the namelist doesn't have that section because someone is not using MPI?

Reply:
I didn't mean we needed to read the decomposition namelist I just meant it was simple and in the spirit of always calling GPTL thereby eliminating  a #IFDEF. I thought the namelistst would always have the decomposition section but I can insert the #IFDEF if you want.